lib/analyzer/binary-version-extractors/openjdk-jre.ts

```diff
@@ -0,0 +1,48 @@
+import { Docker } from '../../docker';
+import { Binary } from '../types';
+
+export {
+  extract,
+  packageNames,
+};
+
+// todo: add common names
+const packageNames = ['java'];
+
+function extract(targetImage: string)
+  : Promise<Binary | null> | null {
+  return new Docker(targetImage).run('java', [ '-version' ])
```

Comment:
Can async here as well?
I thought maybe to extract the new Docker logic as well, but maybe let's wait for to the next binary.